src/StatesDumper/StatesDumper.cs

```diff
@@ -94,7 +102,7 @@ private void OnPersistStorage(DataCache snapshot)
             }
         }
 
-        void IPersistencePlugin.OnCommit(Block block, DataCache snapshot)
+        void IPersistencePlugin.OnCommit(NeoSystem system, Block block, DataCache snapshot)
         {
             if (Settings.Default.PersistAction.HasFlag(PersistActions.StorageChanges))
```

Comment:
```suggestion
            if (system.Settings.Magic != Settings.Default.Network) return;
            if (Settings.Default.PersistAction.HasFlag(PersistActions.StorageChanges))
```

Reply:
We have checked the magic in the `OnPersist` method,  is it still necessary to check again?

Reply:
Should have fixed

Reply:
Yes, already done.